docs/container_runtime.md

```diff
@@ -0,0 +1,75 @@
+# Container Runtimes
+
+## Defaulting
+The machine-controller is able to default to a supported container runtime in case no runtime was specified in the machine-spec.
+Also in case only the version is missing only that will get defaulted.
```

Comment:
What is this sentence supposed to say?

Reply:
You can simply define `machine.Spec.ContainerRuntime.Name` without defining `machine.Spec.ContainerRuntime.Version`

Reply:
I'll clarify that sentence